### ISSUE TITLE
"Export video" does not include captions

### ISSUE THREAD

**Issue by audionerd** [wonderunit/storyboarder#985]
via #978

**Comment by ThatOneAiden** [comment]
THIS. Please, please fix or add this feature soon. Really need it!!

**Comment by WhiteFox88** [comment]
Yes, pleae add this! Need Dialouge printed overlay or a way to print each shot with the dialoge on the side.

**Comment by amedeux** [comment]
I agree on the need for dialogue either printed in overlay or (better) added as text effect/ title track to premiere project, if that's technically doable, that is.
It would be good to have the option to output Action and Notes too.

**Comment by cartoonsailor** [comment]
same. I need this for ease of storyboard exporting and also to preview my timing to potential clients. only sending autoplaying gifs is extremely inconvenient and having the captions on the exported videos would benefit greatly. please consider. :(

**Comment by tin2tin** [comment]
I guess the easiest way is to export a .srt file: https://en.wikipedia.org/wiki/SubRip
And use the ffmpeg to burn in the subtitles: https://trac.ffmpeg.org/wiki/HowToBurnSubtitlesIntoVideo

**Comment by mistafisha** [comment]
Why has this not been implemented yet? It seems like it would be a standard feature, and at least one that is in demand!

**Comment by dannytayara** [comment]
I have this same request. FYI dialogue does export with a gif, but not video yet.

**Comment by seshasaisrivatsav** [comment]
i have the same issue. Please add

**Comment by Kylin-Lau** [comment]
After so many years, will this function not be added? It's really necessary. Please add

**Comment by wenual** [comment]
Is it alive? Just here to mention that someones still waiting :)

**Comment by PrayerVoid** [comment]
I'm still waiting too! :)

**Comment by OctopusEnthusiast** [comment]
Also waiting!
I hope we get a response from the devs!
